src/_css/dark.css

```diff
@@ -0,0 +1,25 @@
+:root {
+  --html-color: var(--nearwhite);
+  --html-background-color: var(--black);
+  --main-color:var(--nearwhite);
+  --main-background-color: var(--darkestgray);
+  --footer-color: var(--nearwhite);
+  --footer-background-color: var(--darkgray);
+  --main-and-footer-link-color: var(--yellow);
+  --avatar-img-filter: grayscale(50%);
+  --icon-no-support-filter: brightness(0.5) grayscale(100%);
+  --note-color: var(--darkestgray);
+  --note-background-color: var(--yellow);
+  --tag-color: var(--nearwhite);
+  --tag-background-color: var(--darkgray);
+  --logo-hover-background-color: var(--darkgray);
+}
+
+:root {
+  -webkit-font-smoothing: antialiased;
```

Comment:
Is this necessary?

Reply:
Yes, I would argue it improves legibility significantly.

With antialiasing:

<img width="734" alt="Screen Shot 2019-10-03 at 18 05 15" src="https://user-images.githubusercontent.com/145676/66144060-75ae6100-e608-11e9-9b00-18c6a3f04ed0.png">

Without antialiasing:

<img width="723" alt="Screen Shot 2019-10-03 at 18 05 28" src="https://user-images.githubusercontent.com/145676/66144079-7b0bab80-e608-11e9-846b-bb4d4ed85434.png">